src/System.Private.Uri/src/System.Private.Uri.csproj

```diff
@@ -84,17 +84,14 @@
     <Compile Include="$(CommonPath)\Interop\Windows\kernel32\Interop.Normalization.cs">
       <Link>Common\Interop\Windows\kernel32\Interop.Normalization.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)\Interop\Windows\normaliz\Interop.Idna.cs">
+    <Compile Include="$(CommonPath)\Interop\Windows\normaliz\Interop.Idna.cs" Condition="'$(TargetGroup)'=='uap101aot'">
```

Comment:
uap101aot is one of the TargetGroup in .builds files. without this we'll get build break. 
ApiCompatBaseline.uap101aot.txt just to make the ApiCompat happy because we moved the contract to System.Runtime and the there is no implementation in corelib yet